apps/generic-issuance-client/.env.example

```diff
@@ -0,0 +1,2 @@
+STYTCH_PUBLIC_TOKEN=
```

Comment:
add comments describing what these are and who on the team to contact to get one